hbase/src/main/java/org/apache/metamodel/hbase/HBaseWriter.java

```diff
@@ -0,0 +1,147 @@
+import org.apache.hadoop.hbase.client.Get;
+import org.apache.hadoop.hbase.client.Put;
+import org.apache.hadoop.hbase.client.Table;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.apache.metamodel.MetaModelException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class HBaseWriter extends Configured {
+
+    private static final Logger logger = LoggerFactory.getLogger(HBaseWriter.class);
+
+    private final Connection _connection;
+
+    public HBaseWriter(Configuration configuration) throws IOException {
+        _connection = ConnectionFactory.createConnection(configuration);
+    }
+
+    public void writeRow(HBaseTable hBaseTable, HBaseColumn[] outputColumns, Object[] values) throws IOException {
+        try {
+            Table table = _connection.getTable(TableName.valueOf(hBaseTable.getName()));
+            try {
+                int indexOfIdColumn = 0;
+                boolean idColumnFound = false;
+                while (!idColumnFound && indexOfIdColumn < outputColumns.length) {
+                    if (outputColumns[indexOfIdColumn].getColumnFamily().equals(HBaseDataContext.FIELD_ID)) {
+                        idColumnFound = true;
+                    } else {
+                        indexOfIdColumn++;
+                    }
+                }
+                if (!idColumnFound) {
+                    throw new MetaModelException("The ID Column family was not found");
+                }
+
+                Put put = new Put(Bytes.toBytes(values[indexOfIdColumn].toString()));
+
+                for (int i = 0; i < outputColumns.length; i++) {
+                    if (!outputColumns[i].getColumnFamily().equals(HBaseDataContext.FIELD_ID)) {
+                        put.addColumn(Bytes.toBytes(outputColumns[i].getColumnFamily()), Bytes.toBytes(outputColumns[i]
+                                .getQualifier()), Bytes.toBytes(values[i].toString()));
+                    }
+                }
+                table.put(put);
+            } finally {
+                table.close();
+            }
+        } finally {
+            _connection.close();
+        }
+    }
+
+    public void deleteRow(HBaseTable hBaseTable, Object key) throws IOException {
+        try {
+            Table table = _connection.getTable(TableName.valueOf(hBaseTable.getName()));
+            try {
+                if (rowExists(table, key) == true) {
+                    table.delete(new Delete(Bytes.toBytes(key.toString())));
+                } else {
+                    logger.warn("Rowkey with value " + key.toString() + " doesn't exist in the table");
+                }
+            } finally {
+                table.close();
+            }
+        } finally {
+            _connection.close();
+        }
+    }
+
+    private boolean rowExists(Table table, Object key) throws IOException {
+        Get get = new Get(Bytes.toBytes(key.toString()));
+        return (table.get(get).isEmpty()) == true ? false : true;
```

Comment:
`table.get(get).isEmpty()` already returns a boolean, so I think you can simplify this (and making it easier to understand) by changing it to 

```
return !table.get(get).isEmpty();
```